index.js

```diff
@@ -32,7 +44,12 @@ module.exports = {
     if (options.liveReload !== true) { return; }
 
     process.env.EMBER_CLI_INJECT_LIVE_RELOAD_PORT = options.liveReloadPort;
-    process.env.EMBER_CLI_INJECT_LIVE_RELOAD_BASEURL = options.baseURL; // default is '/'
+    process.env.EMBER_CLI_INJECT_LIVE_RELOAD_BASEURL = options.baseURL;
+
+    if (options.detectLiveReloadOrigin === true) {
+      process.env.EMBER_CLI_INJECT_LIVE_RELOAD_HOST = options.host;
+      process.env.EMBER_CLI_INJECT_LIVE_RELOAD_ORIGIN = util.format('http://%s:%d/', options.host, options.port);
```

Comment:
I inspected the `options` object but did not see anything that specified what protocol was being used. Is the SSL stuff something that is enabled via the proxy flag (I'm not very familiar with running `ember s` with SSL)?